include/trace/events/intel-sof.h

```diff
@@ -0,0 +1,138 @@
+/* SPDX-License-Identifier: GPL-2.0 */
+#undef TRACE_SYSTEM
+#define TRACE_SYSTEM intel-sof
+
+/*
+ * The TRACE_SYSTEM_VAR defaults to TRACE_SYSTEM, but must be a
+ * legitimate C variable. It is not exported to user space.
+ */
+#undef TRACE_SYSTEM_VAR
+#define TRACE_SYSTEM_VAR intel_sof
+
+#if !defined(_TRACE_INTEL_SOF_H) || defined(TRACE_HEADER_MULTI_READ)
+#define _TRACE_INTEL_SOF_H
+
+#include <linux/types.h>
+#include <linux/ktime.h>
+#include <linux/tracepoint.h>
+
+/* should be the same as rmbox */
+#define TRACE_CLASS_IRQ		(1 << 24)
```

Comment:
We cant decode this here, it must be decoded in userspace. i.e. we should emit a timestamp and then ASCII HEX output of the data (unless it's common data among all trace). Please also see the lates tarce updates on master branch for ABI update.

Reply:
@lgirdwood So the output data will look like "14ff3e 19259f", right? The first element is the timestamp and the second one is the data. Also we need an user space tool to parse these output data? Could you show me the commit id, please. I will look at it.

Reply:
@bardliao rmbox does the parsing. its part of the soft repo